formats/shared/definitions/routes_redirects.jsonnet

```diff
@@ -62,17 +62,7 @@
           "exact",
         ],
       },
-      destination: {
-        type: "string",
-        anyOf: [
-          {
-            "$ref": "#/definitions/absolute_fullpath",
```

Comment:
This might be an enhancement for another time, but I wonder if all other uses of this pattern could be changed to  "uri" as well?

Reply:
Yeah - I'm not sure if they're always validated to the same rigor as this one is via Publishing API. I think for most cases the validation in the schemas is stricter than Publishing API, but I think this case is special as there's either none or very loose schema validation for the unpublishing requests (which share the same redirect validation as this)